README.md

```diff
@@ -1,3 +1,283 @@
 # LavaQueue
 
-yes, no docs lol.
\ No newline at end of file
+> [!IMPORTANT]
+> This plugin *requires* Lavalink `v4.0.5` or greater
+
+A simple queue plugin for [Lavalink](https://github.com/lavalink-devs/Lavalink) with a REST API.
+
+## Summary
+
+* [Installation](#installation)
+* [API](#api)
+* [Get Queue](#get-queue)
+* [Update Queue](#update-queue)
+* [Next Queue Track](#next-queue-track)
+* [Previous Queue Track](#previous-queue-track)
+* [Add Queue Track](#add-queue-tracks)
+* [Update Queue Track](#update-queue-tracks)
+* [Get Queue Track](#get-queue-track)
+* [Add Queue Track](#add-queue-track)
+* [Delete Queue Track(s)](#delete-queue-track(s))
+* [Move Queue Track](#move-queue-track)
+* [Get Queue History](#get-queue-history)
+* [Get Queue History Track](#get-queue-history-track)
+
+## Installation
+
+To install this plugin either download the latest release and place it into your `plugins` folder or add the following into your `application.yml`
+
+Replace x.y.z with the latest version number or short commit hash.
+
+```yaml
+lavalink:
+  plugins:
+    - dependency: "com.github.topi314.lavaqueue:lavaqueue-plugin:x.y.z"
+      snapshot: false # set to true if you want to use snapshot builds (see below)
+```
+
+Snapshot builds are available at https://maven.lavalink.dev/#/snapshots with the short commit hash as the version.
+
+---
+
+### Queue Types
```

Comment:
in https://github.com/topi314/LavaQueue/commit/98c97f9e340361714000fee5f469f6aba7c8ce0b i renamed `queue type` to `queue mode`